com.unity.ml-agents/Runtime/SideChannels/StatsSideChannel.cs

```diff
@@ -0,0 +1,41 @@
+using System;
+namespace MLAgents.SideChannels
+{
+    public class StatsSideChannel : SideChannel
+    {
+        const string k_StatsSideChannelDefaultId = "a1d8f7b7-cec8-50f9-b78b-d3e165a78520";
+
+        /// <summary>
+        /// Initializes the side channel with the provided channel ID.
+        /// </summary>
+        internal StatsSideChannel()
+        {
+            ChannelId = new Guid(k_StatsSideChannelDefaultId);
+        }
+
+        /// <summary>
+        /// Add a stat value for reporting. This will appear in the Tensorboard summary and trainer gauges.
```

Comment:
This feels like great documentation for the class, not just this method.

Reply:
Done